thesis-de/content-de.tex

```diff
@@ -327,6 +327,28 @@ \subsection{Datenerfassungssysteme}
 \subsection{Datenbanksystem / API}
 
 \subsection{Frontend}
+Die Weboberfläche besitzt zwei Schnittstellen, welche aufgeteilt sind, mit der
+Abfrage der Daten an dem Datenhaltungssystem und zum anderen mit der Ausgabe
+und Anzeige für den User bzw. dem Administrator. Eine Schnittstelle auf einem
+Webinterface dient in erster Linie zur Anzeige von Daten, die über ein anderes
+System(BackEnd) erhalten wurden. 
+
+Die erste Schnittstelle der Weboberfläche hat nur eine lesende Berechtigung,
+auf den Datenhaltungssystemen, da keine Änderung durch die Weboberfläche
+durchgeführt werden muss. Die Schnittstelle zwischen den Datenhaltungssystemen
```

Comment:
[Neuer Absatz]
Neben der Verwendung eines Treibers in Verbindung mit einem Berechtigungskonzept, kann auch ein API-System zum Einsatz kommen. Die Vorteile, sowie die Rolle dieses Systems innerhalb dieses Projektes kann aus Punkt~\ref{subsec:schnittstelle_datenbereitstellung} entnommen werden.